app.py

```diff
@@ -189,12 +190,11 @@ def register():
         last_name = request.form['last_name']
         email = request.form['email']
         password = request.form['password']
-
         conn = connect_to_db()
 
         if conn:
             if register_user(conn, username, first_name, last_name, email, password):
-                return "Registration successful", 200  # Return a success response with HTTP status 200
+                return redirect(url_for('user_home')) # Return a success response with HTTP status 200
```

Comment:
It's good that you redirected to home page, it will take user to user home page after registration